functions.py

```diff
@@ -42,42 +44,31 @@ def get_stats(text):
     avg_exc = tokens.count("!") / float(len(sentences))
 
     return {
-        "Mean words per sentence": awps,
-        "Sentence length SD": wpssd,
+        "Words p/s mean": awps,
+        "Words p/s SD": wpssd,
         "Lexical diversity": lex_diversity,
```

Comment:
Yeah, it's meaningless when bootstrapped. I also had concerns regarding its sensitivity in general. I suggest getting rid of it.